src/ui/common/src/reducers/serverConfig.ts

```diff
@@ -0,0 +1,60 @@
+import { createSlice } from '@reduxjs/toolkit';
```

Comment:
Moved the getServerConfig into it's own reducer since it's used throughout the integrations cards now.